pkg/streamingpromql/engine_test.go

```diff
@@ -1396,28 +1396,32 @@ func TestMemoryConsumptionLimit_SingleQueries(t *testing.T) {
 			expr:          "some_metric",
 			shouldSucceed: true,
 
-			// Each series has five samples, which will be rounded up to 8 (the nearest power of two) by the bucketed pool,
+			// Each series has five samples, which will be rounded up to eight (the nearest power of two) by the bucketed pool,
 			// and we have five series and each of the series has SeriesMetadata.
```

Comment:
Maybe this would be clearer?

```suggestion
			// and we have five series and each of the series has labels of the same size.
```

(same applies below)